src/cloudant/database.py

```diff
@@ -489,6 +489,32 @@ def unshare_database(self, username):
         resp.raise_for_status()
         return resp.json()
 
+    def index(self):
```

Comment:
Better call Kenny Loggins, cause your in the Danger Zone... 
This clobbers self.index in the base class, right? search_indices maybe?